### DISCUSSION TITLE
Windows remote login not working as expected

### DISCUSSION THREAD

**Discussion by jcmickey33** [rustdesk/rustdesk#8785] [Q&A] [answered]
BLUF: trying to pass in the login PIN to remote machine is not working

My church had been using TeamViewer for remote access into our AV computer to set things up for the coming week from home. I was asked to help look into alternatives, and found Rustdesk. When running some tests, if I open the client on both machines (rustdesk-1.2.6-x86_64 on each Windows machine) and have the remote machine unlocked already, I'm able to connect just fine and do whatever I want to do.

However, if the remote machine is locked, I'm unable to pass the login PIN through to be able to get to the desktop. I can connect just fine and see the lock screen on the other computer. And (testing on my wife's computer) I can get the PIN field to open, but none of my input goes through.

Below is the menu I'm given from the lightning bolt button
![image](https://github.com/user-attachments/assets/f8a39db5-6fb5-4017-b445-e773d43ba6a3)

I've tried editing the "OS password" field to contain the PIN, but that hasn't worked yet. The remote restart works
![image](https://github.com/user-attachments/assets/9b74715b-4465-49e8-b03d-5cc9fc8277c0)

but upon restarting, I can't connect because the client on the receiver isn't running (I'm sure I could set it to automatically start on boot, but that's not the issue). I've tried both the right keypad and the "regular" number keys. I've tried adjusting all of the keyboard settings. I have the remote machine set to "Full access" on permissions. And under General > Other, I've got the following
![image](https://github.com/user-attachments/assets/831e2a0b-2734-440d-81c3-917157524d1c)

I'm sure (I'm hopeful) it's just something small I'm missing, but any help or suggestions you have would be welcome! Thanks.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#windows-uac

**Reply by jcmickey33** [reply]
But if I'm connecting to a remote device that no one's sitting at, I can't accept the elevation. And if I can click that as the remote user, that's cool, but the issue is that I can't input the PIN for the remote machine (i.e., I can't even login to the machine)

**Reply by jcmickey33** [reply, marked as answer]
Even though there was no context in your response and not really any direct indicator in the FAQ, I'll go ahead and assume what you meant was:

"If you run it as an administrator on the remote side, you'll be able to pass keystrokes through" 

That did work
